docs/dev_process.md

```diff
@@ -23,17 +23,32 @@ The repo includes a simple test application that can be run locally to develop t
 First, clone the repo:
 
 ```sh
-git clone [repo url]
-cd django-pattern-library
+$ git clone [repo url]
+$ cd django-pattern-library
 ```
 
+
+### Run a local build without docker
 We use [Poetry](https://poetry.eustace.io/docs/) to manage Python dependencies, so make sure you've got it installed.
 
 Then you can install the dependencies and run the test app:
 
 ```sh
-poetry install  # installs the library and its dependencies in editable mode
-poetry run ./runserver.sh  # runs the test app using the Django development server
+$ poetry install  # installs the library and its dependencies in editable mode
+$ poetry run ./runserver.sh  # runs the test app using the Django development server
```

Comment:
I see the `runserver.sh` script has been removed, but it’s still mentioned here. Do we want to remove it here and inline its content instead?

```suggestion
$ poetry run 'django-admin runserver --settings=tests.settings --pythonpath=.'  # runs the test app using the Django development server
```